plugins/rhs_plugin/index.js

```diff
@@ -12,7 +12,8 @@ function mapStateToProps(state) {
     const rhsPlugins = state.plugins.components.RightHandSidebarComponent;
     const pluginId = getPluginId(state);
 
-    const pluginName = rhsPlugins.find((element) => element.id === pluginId).title;
+    const plugin = rhsPlugins.find((element) => element.id === pluginId);
+    const pluginName = plugin ? plugin.title : '';
```

Comment:
This looks good but just wondering why it was trying to initialize the RHS of the plugin that was uninstalled🤔

Reply:
Agreed, I'm not sure what the race condition is here.